### PR TITLE
[2.6.3] pass typers to norman stores

### PR DESCRIPTION
**Pull request by paynejacob**
rancher/rancher#34721
rancher/rancher#30718
rancher/rancher#25487

See https://github.com/rancher/norman/pull/402 for details

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -166,3 +166,5 @@ require (
 	sigs.k8s.io/controller-runtime v0.9.0-beta.0
 	sigs.k8s.io/yaml v1.2.0
 )
+
+replace github.com/rancher/norman => github.com/paynejacob/norman v0.0.0-20210921222300-55bf1baf0ad7
```
Comment (paynejacob) [inline go.mod:170]: Temporary to allow for testing

File: go.mod [hunk 2]
```diff
@@ -166,3 +166,5 @@ require (
 	sigs.k8s.io/controller-runtime v0.9.0-beta.0
 	sigs.k8s.io/yaml v1.2.0
 )
+
+replace github.com/rancher/norman => github.com/paynejacob/norman v0.0.0-20211001225425-d7812fc743ce
```
Comment (paynejacob) [inline go.mod:170]: waiting on https://github.com/rancher/norman/pull/402

File: pkg/api/norman/customization/catalog/store.go [hunk 3]
```diff
@@ -5,6 +5,8 @@ import (
 	"fmt"
 	"strconv"
 
+	"github.com/rancher/rancher/pkg/api/scheme"
+
```
Comment (paynejacob) [inline pkg/api/norman/customization/catalog/store.go:9]: todo @paynejacob

**Review by thedadams — APPROVED** [review]

**Review by cbron — APPROVED** [review]

**Comment by paynejacob** [comment]
@thedadams @snasovich @rmweir I had to change the resources, the inline json tag fixed the nested template object but it would overwrite the object meta fields with blank ones causing issues with the schema changes.  I tested these changes and it does not break anything because as far as the crd is concerned those fields were always there so the data did not change.

**Review by thedadams — APPROVED** [review]

**Review by cbron — APPROVED** [review]

**Pull request merged**